src/frontend/apps/desk/src/features/teams/components/Member/ModalRole.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Modal,
+  ModalSize,
+  Radio,
+  RadioGroup,
+  VariantType,
+  useToastProvider,
+} from '@openfun/cunningham-react';
+import { t } from 'i18next';
+import { useState } from 'react';
+
+import { Box, Text, TextErrors } from '@/components';
+import { useAuthStore } from '@/features/auth';
+import { Access, Role, useUpdateTeamAccess } from '@/features/teams/api/';
+
+interface ModalRoleProps {
+  access: Access;
+  onClose: () => void;
+  teamId: string;
+}
+
+export const ModalRole = ({ onClose, teamId, access }: ModalRoleProps) => {
```

Comment:
it might be a Cunningham issue, can we allow closing the modal on click outside?